scripts/balancing.lua

```diff
@@ -1980,17 +1980,17 @@ DefineAnimations(
 )
 
 for i,spec in ipairs({
-   { Var = "Supply", Gold = 700, Unit = "brigand" },
-   { Var = "Demand", Gold = 1200, Unit = "ogre" },
+   { Var = "Supply", Gold = 1000, Wood = 4000, Unit = "brigand" }, --wood is used for cooldown. So long to reduce micro.
```

Comment:
Calling this `Wood` doesn't make sense.

Reply:
i thought we need to use some existing variables, didnt know i could use any name